client/src/components/CodeOfConduct.jsx

```diff
@@ -1,79 +1,123 @@
 import { useState } from 'react';
 import PropTypes from 'prop-types';
-import StyledFieldset from './StyledFieldset';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import { FormControlLabel } from '@mui/material';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import Button from '@mui/material/Button';
+import RadioGroup from '@mui/material/RadioGroup';
+import Typography from '@mui/material/Typography';
+import ErrorIcon from '@mui/icons-material/Error';
+import Radio from '@mui/material/Radio';
+import MUIFieldsetStyles from './MUIStyledFieldSet';
+
+
 
 export default function CodeOfConduct({ setRegisterStep }) {
   const [accepted, setAccepted] = useState(false);
 
   return (
-    <StyledFieldset>
-      <legend>Code of Conduct</legend>
-      <p className="instructions">Review our code of conduct</p>
-      <p>Please review the code of conduct</p>
+    <MUIFieldsetStyles>
```

Comment:
Can you add some padding to the purple box? The text is up pretty tight against the borders.